tests/testMap.cpp

```diff
@@ -19,6 +16,30 @@ BOOST_AUTO_TEST_CASE(myTestCase){
     token.push_back("working");
     token.push_back("okay");
 
-    BOOST_TEST(token == myMap.tokenize("This is a test -- is the code wor-king okay?"));
+    BOOST_TEST(token == Map::tokenize("This is a test -- is the code wor-king okay?"));
+    BOOST_TEST(true);
+}
+
+BOOST_AUTO_TEST_CASE(TestMap_map){
+
+    size_t bufSize = 1024;
+    size_t lineCount = 2;
+    size_t currLine = 1;
+    std::string inputFile = "tests/workdir/input/testMap.txt";
+
+    Map myMap = Map(inputFile, bufSize, lineCount);
+
+    std::vector<std::string> fileLines;
+    std::string testLine = "Hello Map Class!";
+    fileLines.push_back(testLine);
+    fileManager->writeFile(fileManager->getInputDirectory(), "testMap.txt", fileLines);
+
+    myMap.map(testLine, currLine);
```

Comment:
The reason the test case doesn't throw an error is because of this `.map()` function here, which disregards the instance's input file altogether and processes the `testLine` instead, which I don't think was the intended behavior of the function/specification.

Reply:
I decided to pass the filename into the constructor and have it as a class member variable because we're creating a separate instance of the Map class for each filename and the filename is needed in multiple member functions. Regarding the test, I may have skipped the step of reading the contents of the "testMap.txt" file that I create because I already knew what was in it. But if I add that part, I can then pass in the line that was read in to the map() method and it will better align with the behavior that we want. How does that sound?